menuinst/platforms/base.py

```diff
@@ -37,7 +37,10 @@ def __init__(
         self.prefix = Path(prefix)
         self.base_prefix = Path(base_prefix)
 
-        self.env_name = None
+        if self.prefix.samefile(self.base_prefix):
+            self.env_name = None
```

Comment:
That's actually quite tricky. "base environment" is a concept, that doesn't mean that the base environment has to be called "base" in all implementations. On the other hand, `self.base_prefix.name` is incorrect because that's the directory name of the installation, not the name of the environment.

Reply:
Since we already have the installation directory name in https://github.com/conda/menuinst/blob/f3c25ed1f9a825836ca3980d4e445db3ff0b0781/menuinst/platforms/base.py#L72, so I’d say we go for `base`.